Source/Core/DolphinWX/Main.cpp

```diff
@@ -141,6 +141,8 @@ bool DolphinApp::OnInit()
 
 	if (m_select_video_backend && !m_video_backend_name.empty())
 		SConfig::GetInstance().m_strVideoBackend = WxStrToStr(m_video_backend_name);
+	if (m_select_slippi_port && m_slippi_port > 0 && m_slippi_port < 65536)
```

Comment:
```suggestion
	if (m_select_slippi_port && m_slippi_port > 1024 && m_slippi_port < 65536)
```

i just remembered that on linux you can't bind to a port under 1025 unless you're root. don't have a strong opinion on if we should ifdef it or apply it everywhere

Reply:
I'm seeing <1024, is that ok?

Reply:
yeah you're right, i misread it as 1-1024